Mac/BuildScript/build-installer.py

```diff
@@ -1490,7 +1490,7 @@ def packageFromRecipe(targetDir, recipe):
                 IFPkgFlagRelocatable=False,
                 IFPkgFlagRestartAction="NoRestart",
                 IFPkgFlagRootVolumeOnly=True,
-                IFPkgFlagUpdateInstalledLangauges=False,
+                IFPkgFlagUpdateInstalledLanguages=False,
```

Comment:
That particular typo is in a data structure that is no longer used in installer builds and will eventually be removed. For simplicity, I would just ignore it.

Reply:
This looks like a bug fix to me.  If someone could demonstrate undesired behavior caused by this typo, it would warrant a bug report and a separate PR.

Reply:
FTR, Ned responded here: https://github.com/python/cpython/pull/101868#pullrequestreview-1383988443